MAINTAINERS.md

```diff
@@ -0,0 +1,44 @@
+# OpenTelemetry Go Compile Instrumentation
+
+- [Overview](#overview)
+- [Maintainers](#maintainers)
+- [Approvers](#approvers)
+- [Triagers](#triagers)
+- [Emeritus Maintainers](#emeritus-maintainers)
+
+## Overview
+
+This document contains a list of maintainers in this repo. See [opensearch-project/.github/RESPONSIBILITIES.md](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#maintainer-responsibilities) that explains what the role of maintainer means, what maintainers do in this and other repos, and how they should be doing it. If you're interested in contributing, and becoming a maintainer, see [CONTRIBUTING](CONTRIBUTING.md).
+
+## Maintainers
+
+| Name               | GitHub ID                                         | Affiliation |
+| ------------------ | ------------------------------------------------- | ----------- |
+| Huxing Zhang       | [ralf0131](https://github.com/ralf0131)           | Alibaba     |
```

Comment:
I remember when the SIG was initially set up, there should only be three initial maintainers: Romain, Przemyslaw and Huxing. May be we should move us into Approvers section?

Reply:
What do you think of making everyone maintainers?

Reply:
I agree.

Reply:
I'm ok with that, however not sure if we are not going from one extreme to the other.

Reply:
Let's start with this because we need to start from somewhere and we will include more people depending on their contributions.